alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/processing/ProcessorService.java

```diff
@@ -79,11 +79,9 @@ private void onStop() {
     private void executeInternal() {
         Set<NodeRef> nodesToProcess = getNextNodesInTransaction();
         while (!nodesToProcess.isEmpty()) {
-            updateIndexingProgress();
             this.processNodeBatch(nodesToProcess);
             nodesToProcess = getNextNodesInTransaction();
         }
-        updateIndexingProgress();
```

Comment:
Is the removal of the update call outside the loop deliberate?

Reply:
Yes, it was. But now I see that it was also incorrect, because the progress would not get updated anymore after the last batch of transactions was fetched.